src/components/pages/Mentor/MentorDashboard.js

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link, NavLink, Route, Switch } from 'react-router-dom';
+import StudentSearch from '../Student/StudentSearch';
+
+import MentorProfile from './MentorProfile';
+import MentorHome from './MentorHome';
+
+import { fetchMentors } from '../../../state/actions';
+import Logout from '../../Logout.js';
+import { Layout, Menu, PageHeader, Button, Avatar } from 'antd';
+import {
+  HomeOutlined,
+  UserOutlined,
+  //   FormOutlined,
+  LogoutOutlined,
+} from '@ant-design/icons';
+
+const MentorDashboard = props => {
+  const { profile } = props;
+
+  useEffect(() => {
+    props.fetchMentors(1); // change this later with login
+  }, []);
+  console.log('Mentors:', profile);
```

Comment:
Just a reminder they want us to remove all of our console.log calls that we used for testing purposes so the calls aren't in production code.

Reply:
Thanks for the reminder I will remove them.